example/dev.auto.tfvars

```diff
@@ -32,19 +31,5 @@ map_additional_iam_roles = [
     rolearn  = "arn:aws:iam::757583164619:role/sourcefuse-poc-2-admin-role"
   }
 ]
-vpc_name = "refarchdevops-dev-vpc"
-private_subnet_names = [
-  "refarchdevops-dev-privatesubnet-private-us-east-1a",
-  "refarchdevops-dev-privatesubnet-private-us-east-1b"
-]
-public_subnet_names = [
-  "refarchdevops-dev-publicsubnet-public-us-east-1a",
-  "refarchdevops-dev-publicsubnet-public-us-east-1b"
-]
-tags = {
-  Environment = "dev"
-  ENV         = "dev"
-  Project     = "sf-ref-arch"
-  Creator     = "terraform"
-}
-health_check_domains = ["healthcheck-example.sfrefarch.com"]
+vpc_name             = "arc-poc-vpc"
```

Comment:
we can fetch the vpc name from data lookups

Reply:
removed it from tfvars